go/gilded-rose_test.go

```diff
@@ -1,15 +1,134 @@
 package main
 
-import "testing"
+import (
+	"testing"
+)
+
+func LoopDays(items []*Item, days int) {
```

Comment:
Im not convinced by the need to test the effect of several days worth of quality updates. This can be covered by choosing the right single day test cases from reading the spec.
I know others have gone for the several day testing thing so I could be in for a kicking in the discussion.